docs/development/testguide.rst

```diff
@@ -110,6 +209,13 @@ Astropy Test Function
 
 .. autofunction:: astropy.test
 
+pytest
+======
+
+The test suite can be run directly from the native ``pytest`` command. In this
+case, it is important for developers to be aware that they must manually
+rebuild any extensions by running ``setup.py build_ext`` before testing.
```

Comment:
I *think* `setup.py test` automatically runs `doctest` as well (does it still?), but that won't be the case here, right? We need to document any behavior difference, if applicable.

Reply:
Yes, good point. It also supplies `--doctest-rst` by default.